Sources/Plugins/Readers/RayTracer.Png.Reader/RayTracer.Png.Reader.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
I personally don't use implicit usings anywhere, we don't have it in any project, let's stick to this code style everywhere, what do you think?